### ISSUE TITLE
Allow blacklist of repositories and organizations

### ISSUE THREAD

**Issue by crutchcorn** [abusesa/github-backup#9]
Currently, I'm tied into many organizations that I would not like to pull code from in a personal backup.

It would be nice to be able to prevent these repos/orgs from being backed up into this system

**Comment by henri** [comment]
Yes this would be an awesome feature! Also, thanks for creating this excellent script!

**Comment by henri** [comment]
If you only want to pull your personal repositories that should be possible take a look at the example config file section in the ReadMe. But really like the idea of being able specify a list of repositories which are not included in the backup. IMHO the way to go rather than having to add repositories which you would like to include.

**Comment by henri** [comment]
Put in a pull request with basic support for excluding a repository. Only tested on user repository. Further testing is welcome [from the fork](https://github.com/henri/github-backup) prior to accepting the pull request. 

[Link to the pull request](https://github.com/abusesa/github-backup/pull/11) if you have access to approve
